pkg/Library/Dom/container-layout.js

```diff
@@ -60,6 +60,7 @@ export class ContainerLayout extends DragDrop {
     const rectsJson = JSON.stringify(rects);
     if (state.selectedJson !== selectedJson || rectsJson !== rectsJson) {
       state.selectedJson = selectedJson;
+      state.rects = rects;
```

Comment:
Use `deepEqual` instead of JSON testing whenever possible

Reply:
If one of these clauses is more likely to fail, put it first :) Maybe you already did, just mentioning a useful rule of thumb.